app/src/main/res/values-de/strings.xml

```diff
@@ -190,6 +190,7 @@
     <string name="website_title">Website</string>
     <string name="website_encouragement">Besuche die NewPipe Website für weitere Informationen und Neuigkeiten.</string>
     <string name="donation_encouragement">NewPipe wird von Freiwilligen entwickelt, die ihre Freizeit dafür verwenden, dir die beste Nutzererfahrung zu bieten. Gib etwas zurück, indem du den Entwicklern hilfst, NewPipe noch weiter zu verbessern, während sie sich an einer Tasse Kaffee erfreuen.</string>
+    <string name="donation_thank_you">Vielen Dank für deine Spende!</string>
```

Comment:
This just causes conflicts, translate it from Weblate after the PR is merged

Reply:
Weblate can’t deal with this?

Reply:
Not always